### PR TITLE
Adding a rule to detect WriteHijack DLL exploitation by PowerUp

### PR DESCRIPTION
**Pull request by pbssubhash**
Dear Team, 

Kindly review my PR. Please oblige if anything's wrong/missing as this is my first commit to Sigma :-D

### CODE REVIEW

**Comment by pbssubhash** [comment]
Also, I'm not sure about the file.path field. Please guide me with that.. My Kibana Query is this: 
`(process.executable.keyword:*\\powershell.exe AND file.path.keyword:*.bat)` .. I'm not sure if `file.path` gets translated to all platforms. Hence need your support on where can I find the translation to the fields... 

Thanks again.

File: rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: Powerup Write Hijack DLL detection
+id: 602a1f13-c640-4d73-b053-be9a2fa58b96
+status: experimental
+description: Powerup tool's Write Hijack DLL exploits DLL hijacking for privilege escalation. In it's default mode, it builds a self deleting .bat file which executes malicious command. The detection rule relies on creation of the malicious bat file (debug.bat by default). Reference: https://powersploit.readthedocs.io/en/latest/Privesc/Write-HijackDll/
+author: pbssubhash
+date: 2021/08/21
+modified: 2021/08/21
+tags:
```
Comment (frack113) [inline rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml:8]: Hi no need for modified as it is a first release

File: rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - attack.privilege_escalation
+    - attack.defense_evasion
+    - attack.t1574.001
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\powershell.exe'
+        File.path|endswith: '.bat'
```
Comment (frack113) [inline rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml:19]: Use the original field name thanks

Comment (pbssubhash) [reply rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml:19]: The field according to the event log would be `TargetFileName`. However, when I use sigmac to convert the rule, it doesn't convert properly. Is it okay?
image

File: rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml (hunk 1 above)
Comment (pbssubhash) [reply rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml:8]: Noted.

File: rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml (hunk 2 above)
Comment (pbssubhash) [reply rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml]: Modified as advised. kindly check, thanks.

**Comment by frack113** [comment]
Fix the title. Non need to put détection as it is rule to detect . 
So for my typing . On phone in a car it not easy...

**Comment by pbssubhash** [comment]
Modified the title. 
Thanks

**Review by frack113 — APPROVED** [review]
I have just cleanup description and references

**Pull request merged**